<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskInputDialog</class>
 <widget class="QDialog" name="TaskInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="taskLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Task:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="priorityComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="priorityLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>TaskInputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>TaskInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
